package-lock.json

```
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "node-red-contrib-quantum",
       "version": "0.0.1",
```

```suggestion
      "name": "node-red-contrib-quantum",
      "version": "0.0.1",
```
Did you intend to remove this? I ran `npm i --package-lock-only` and it re-added it, so I think it must be required.

---

That is an error, I did not intend to remove it. Thank you !